oauthproxy_test.go

```diff
@@ -1218,7 +1254,7 @@ func TestAuthSkippedForPreflightRequests(t *testing.T) {
 	}))
 	defer upstream.Close()
 
-	opts := options.NewOptions()
+	opts := baseTestOptions()
 	opts.Upstreams = append(opts.Upstreams, upstream.URL)
 	opts.ClientID = "aljsal"
```

Comment:
These aren't needed since the `baseTestOptions` sets alternatives and it looks like we aren't asserting a future value is set to these in any of these tests.

I think this applies to most of these cases with lingering ClientID, ClientSecret, & Cookie.Secrets